tests/semanticdb/expect/Synthetic.expect.scala

```diff
@@ -30,7 +30,7 @@ class Synthetic/*<-example::Synthetic#*/ {
     null.asInstanceOf/*->scala::Any#asInstanceOf().*/[Int/*->scala::Int#*/ => Int/*->scala::Int#*/]/*->scala::Function1#apply().*/(2)
   }
 
-  class J/*<-example::Synthetic#J#*/[T/*<-example::Synthetic#J#[T]*/: Manifest/*->scala::Predef.Manifest#*//*->example::Synthetic#J#[T]*/] { val arr/*<-example::Synthetic#J#arr.*/ = Array/*->scala::Array.*/.empty/*->scala::Array.empty().*/[T/*->example::Synthetic#J#[T]*/] }
+  class J/*<-example::Synthetic#J#*/[T/*<-example::Synthetic#J#[T]*/: Manifest/*->scala::Predef.Manifest#*//*->example::Synthetic#J#[T]*/] { val arr/*<-example::Synthetic#J#arr.*/ = Array/*->scala::Array.*/.empty/*->scala::Array.empty().*/[T/*->example::Synthetic#J#[T]*/]/*->example::Synthetic#J#evidence$1.*/ }
```

Comment:
`/*->example::Synthetic#J#evidence$1.*/ ` should not be added, did I skip comments disussing it?

Reply:
Excluded the symbols that don't present in source from symbol occurrences: https://github.com/lampepfl/dotty/pull/12885/commits/fd3b71ca620c7e40dd36a004e614e49614219b44
In addition to `evidence` occurrence, the commit removed some occurrences but it matches Scala2

Reply:
Followup: recover the symbol occurrence for package object https://github.com/lampepfl/dotty/pull/12885/commits/4f60d9f1828dc68cee40192993280e6af987950d